docs/user_guide/vectors_and_matrices.mdx

```diff
@@ -501,3 +501,69 @@ compile-time.
 ![Fixed resizing](/img/resizing.svg)
 
 </div>
+
+## Matrix Macros 
+### Macros 
+#### dmatrix!
+Construct a dynamic matrix directly from data. The syntax is exactly the same as for matrix!, but instead of producing instances of SMatrix, it produces instances of DMatrix. 
+At the moment it is not usable in `const fn` contexts.
+```rust
+use nalgebra::dmatrix;
+let a = dmatrix![ 1, 2, 3;                  
+                  4, 5, 6;                  
+                  7, 8, 9];
+```
+#### dvector! 
+Construct a dynamic column vector directly from data.
+The syntax is exactly the same as for vector!, but instead of producing instances of SVector, it produces instances of DVector. At the moment it is not usable in `const fn` contexts.
+```rust
+use nalgebra::dmatrix;
+let a = dvector![ 1, 2, 3];
+```
+#### matrix!
+Construct a fixed-size matrix directly from data. 
+This macro facilitates easy construction of matrices when the entries of the matrix are known (either as constants or expressions). 
+This macro produces an instance of SMatrix. This means that the data of the matrix is stored on the stack, and its dimensions are fixed at compile-time. If you want to construct a dynamic matrix, use dmatrix! instead.
+matrix! is intended to be both the simplest and most efficient way to construct (small) matrices, and can also be used in `const fncontexts`.
+The syntax is MATLAB-like. Column elements are separated by a comma (,), and a semi-colon (;) designates that a new row begins.
+```rust
+use nalgebra::matrix;
+// Produces a Matrix3<_> == SMatrix<_, 3, 3>
+let a = matrix![ 1, 2, 3;
+                4, 5, 6;
```

Comment:
This is not properly aligned

Reply:
Updated: 

I updated the spacing between paragraphs to try and match the rest of the document. Hopefully I understood correctly.

Reply:
Sorry, I was not clear: I meant that the *matrix elements* are not aligned, e.g. the `4` appears one character to the left of `1`, which looks odd. Please remove the new line after the comment again!